src/main.ts

```diff
@@ -587,7 +599,7 @@ export default class WhatsApp {
 
   private async getKeys() {
     return new Promise((resolve, reject) => {
-      readFile(this.keysPath!, "utf-8", (err, data) => {
+      readFile(this.parameters.keysPath!, "utf-8", (err, data) => {
```

Comment:
```suggestion
      readFile(this.opts.keysPath!, "utf-8", (err, data) => {
```
Update property again